### ISSUE TITLE
Install error and fix

### ISSUE THREAD

**Issue by ixpress** [swoole/swoole-src#187]
While I was install swoole on my mac,
I tried this ```sudo pecl install swoole```,
I got ```fatal error: 'pcre.h' file not found``` first.
To slove this, I tried ```brew install pcre```.
Then I got this ```fatal error: 'swoole.h' file not found```.
While, what interesting is, when I tried ```make``` from source, it passed!
So, when you got error on install, try this.

**Comment by ichiriac** [comment]
:+1: got the same error with the latest `stable` version swoole-1.7.9 version from PECL :

```
/tmp/pear/temp/swoole/src/core/Base.c:17:20: fatal error: swoole.h: No such file or directory
compilation terminated.
make: *** [src/core/Base.lo] Error 1
ERROR: `make' failed
```

How could you call it stable, it's not even install correctly on a debian 7 distrib !

**Comment by matyhtf** [comment]
This is a bug, fixed in 1.7.10.

You can do:
```
pecl download swoole
tar xvf swoole-1.7.9.tgz
cd swoole-1.7.9
phpize
./configure
make && make install
```

**Comment by noname007** [comment]
i meet the question when install 1.7.15 using pecl.it looks like lack of the lib
ref this 
http://stackoverflow.com/questions/22555561/error-building-fatal-error-pcre-h-no-such-file-or-directory
